screwdriver/api/api.go

```diff
@@ -141,20 +141,28 @@ func writeMultipartYaml(writer *multipart.Writer, specPath string, commandSpec *
 }
 
 func writeMultipartBin(writer *multipart.Writer, commandSpec *util.CommandSpec) error {
```

Comment:
You do not need fix now. But someday I think it is better to make these writeMultipart* and specToPayloadBuf function as method of util.CommandSpec, because these function have very strong relationship with util.CommandSpec and to me, it make sense that util.CommandSpec have these methods.